app/pageutils.py

```diff
@@ -31,6 +31,6 @@ def result_message(search_range, total, title):
 
   range_high = search_range.get('rangeHigh', 0)
   range_low = search_range.get('rangeLow', 0)
-  max_i = max(total, range_high)
+  max_i = min(total, range_high)
   min_i = min(total, range_low)
```

Comment:
If this shouldn't be the max, it should probably not be named max_i. We could call it `upper`, and call min_i `lower`.

Reply:
Have renamed